app/controllers/facility_estimates_controller.rb

```diff
@@ -99,6 +99,40 @@ def recalculate
     redirect_to facility_estimate_path(current_facility, @estimate)
   end
 
+  def duplicate
+    duplicated_estimate = nil
+
+    Estimate.transaction do
+
+      duplicated_estimate = @estimate.dup
```

Comment:
should we set the name to `(Copy) #{@estimate.name}` or something like that to let know users know they are viewing the copy?